src/main/java/net/sf/jabref/model/cleanup/FieldFormatterCleanups.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.model.cleanup;
```

Comment:
Mhhh.... none of the other metadata items are currently under the metadata namespace:
import net.sf.jabref.model.cleanup.FieldFormatterCleanups;
import net.sf.jabref.model.bibtexkeypattern.AbstractBibtexKeyPattern;
import net.sf.jabref.model.database.BibDatabaseMode;
import net.sf.jabref.model.groups.GroupTreeNode;

I think it is more consistent to have a model.cleanup or also move the rest to model.metadata(.groups/.bibtexkeypattern...)